test/unit/app.spec.js

```diff
@@ -37,6 +37,26 @@ describe('App', function() {
     });
   });
 
+  describe('when restarting the app', function() {
```

Comment:
honestly this test should go in the app-lifecycle spec and should resemble on of those:  https://github.com/RoundingWellOS/marionette.toolkit/blob/c91fd3f289717264c73e6286f10a1f45c97b8f29/test/unit/app-lifecycle.spec.js#L23